packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2832,6 +2832,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             throw new UsageError("can't get state when offline load disabled");
         }
 
+        if (this._orderSequentiallyCalls !== 0) {
```

Comment:
FYI - you will have conflict with my changes in main. Not sure when it shows up in next though.
That said, your change is good, and we should merge both of our changes (my this.flush() call and your change).